cms/envs/common.py

```diff
@@ -37,7 +37,6 @@
 
 # We intentionally define lots of variables that aren't used, and
 # want to import all variables from base settings files
-# pylint: disable=unused-import
```

Comment:
Switching to this instead should get rid of all warnings in this file (after deleting a duplicate disabling of `wrong-import-order` further down in the file):

```
# pylint: disable=unused-import, useless-suppression, wrong-import-order, wrong-import-position
# pylint: enable=useless-suppression
```

Reply:
done